benchmark/server/scenarios/system.js

```diff
@@ -12,57 +11,31 @@ const suite = new Benchmark.Suite('system', {
 });
 Benchmark.options.minSamples = 100;
 
-const materialSystem = compose(palette, spacing, typography);
-const styledSystem = compose2(color, space, fontFamily, fontSize);
-
 const materialSystemTheme = createMuiTheme();
 
-const styledSystemTheme = createMuiTheme();
-styledSystemTheme.breakpoints = ['40em', '52em', '64em'];
-styledSystemTheme.colors = styledSystemTheme.palette;
-styledSystemTheme.fontSizes = styledSystemTheme.typography;
-styledSystemTheme.fonts = styledSystemTheme.typography;
+const styledSystemTheme = {
+  breakpoints: ['40em', '52em', '64em'],
+  colors: {
+    primary: materialSystemTheme.palette.primary,
+    background: materialSystemTheme.palette.background,
+  },
+  fontSizes: materialSystemTheme.typography,
+  fonts: materialSystemTheme.typography,
+};
+
+const styledSystemCSSFunction = css({
```

Comment:
I'd say this is the equivalent based on the result. Invoking only `css({...})` is resulting in just a function, we need to invoke that function to get to the style object. Not sure why I needed to repeat the style input on both places, either I am doing something wrong, or this is not intended to be used like this...